src/node_crypto_common-inl.h

```diff
@@ -0,0 +1,423 @@
+#ifndef SRC_NODE_CRYPTO_COMMON_INL_H_
+#define SRC_NODE_CRYPTO_COMMON_INL_H_
+
+#include "env-inl.h"
+#include "node_crypto.h"
+#include "node_crypto_common.h"
+#include "node.h"
+#include "node_internals.h"
+#include "node_url.h"
+#include "string_bytes.h"
+#include "v8.h"
+
+#include <openssl/ssl.h>
+#include <openssl/x509v3.h>
+
+#include <string>
+#include <unordered_map>
+
+namespace node {
+
+using v8::Array;
+using v8::Context;
+using v8::Integer;
+using v8::Isolate;
+using v8::Local;
+using v8::Object;
+using v8::Value;
+
+namespace crypto {
+
+inline void LogSecret(
```

Comment:
Ditto as in the other PR, `inline` should typically go into the `.h` header file; that way, the compiler knows that the method should be declared inline and warns when it’s used but not defined. The other way around, it’s just the linker that fails, and not necessarily reliably.

That being said, it probably makes sense for any functions here that’s longer than 2 lines to be defined in a `.cc` file.

Reply:
Since most of the functions in the inl.h are longer than 2 line, I just moved the entire lot in a cc